.github/workflows/create_release.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           zip -r -j ${ARCHIVE} ${BUILD_DIR}
 
       - name: Save artifacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
Necessary, due to:

```
Error: This request has been automatically failed because it uses a deprecated version of `actions/download-artifact: v2`. Learn more: https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/
```